layouts/index.html

```diff
@@ -30,6 +30,9 @@ <h3>Sponsorship</h3>
                 </a>
             </div>
             <script src='https://opencollective.com/bcneng/banner.js?style={"a":{"color":"black", "fontFamily":"sans-serif"},"h2":{"fontFamily":"sans-serif","fontWeight":"bold","fontSize":"1.2em"}}'></script>
+            <div class="sponsor-logos">
+                <img src="/images/sponsor-logos/donut.png" class="img-fluid sponsor-logo-image" alt="Donut logo">
```

Comment:
What do you think about making it link to the donut homepage?

Reply:
Good idea! Updated!

Reply:
Sorry to nitpick, but it would be great to center it in mobile:

![image](https://user-images.githubusercontent.com/349328/208996710-8b7b3a0b-3c6e-43c3-aa8a-7442f25b00a4.png)

Reply:
You know what: forget it. I'll add a h2 or h3 header tomorrow above it.